### DISCUSSION TITLE
Standard File - What happened?

### DISCUSSION THREAD

**Discussion by BobTB** [standardnotes/forum#2719] [General] [answered]
What happened with [Standard File](https://github.com/standardfile)?  Is this project dead? I am using it in a docker container now for years, and am wondering if this is still a thing?

**Comment by JaspalSuri** [comment, marked as answer]
Hi @BobTB,

As @TheodoreChu pointed out on Slack, this project has been deprecated. There is an **unofficial** project based off of it that might interest you, known as [SN-FS](https://github.com/tannercollin/standardnotes-fs).
